score/README.md

```diff
@@ -1,5 +1,149 @@
+- [Miscellaneous](#miscellaneous)
+
+<!-- /TOC -->
+
+</details>
+
+## About this application
+
+This application is used to compare experimental versions of the Justice40 score to established environmental justice indices, such as EJSCREEN, CalEnviroScreen, and so on. The goal of this comparison is to identify the
+
+### Score comparison workflow
+
+#### Workflow Diagram
+
+TODO add mermaid diagram
+
+#### Step 1: Run the ETL script for each data source
+
+1. Call the `etl-run` command using the application manager `application.py`
+   - With Docker: `docker run --rm -it j40_score /bin/sh -c "python3 application.py etl-run"`
+   - With Poetry: `poetry run python application.py etl-run`
+1. The `etl-run` command will execute the corresponding ETL script for each data source in `etl/sources/`. For example, `etl/sources/ejscreen/etl.py` is the ETL script for EJSCREEN data.
+1. Each ETL script will extract the data from its original source, then format the data into `.csv` files that get stored in the relevant folder in `data/dataset/`. For example, HUD Housing data is stored in `data/dataset/hud_housing/usa.csv`
+
+>**NOTE:** You have the option to pass the name of a specific data source to the `etl-run` command, which will limit the execution of the ETL process to that specific data source.
+> For example: `poetry run python application.py etl-run ejscreen` would only run the ETL process for EJSCREEN data.
+
+#### Step 2: Calculate the Justice40 scores
+
+1. Call the `score-run` command using the application manager `application.py`
+   - With Docker: `docker run --rm -it j40_score /bin/sh -c "python3 application.py score-run"`
+   - With Poetry: `poetry run python application.py score-run`
+1. The `score-run` command will execute the `etl/score/etl.py` script which loads the data from each of the source files added to the `data/dataset/` directory by the ETL scripts in Step 1.
+1. These data sets are merged into a single dataframe using their Census Block Group GEOID as a common key, and the data in each of the columns is standardized in two ways:
+    - Their [percentile rank](https://en.wikipedia.org/wiki/Percentile_rank) is calculated, which tells us what percentage of other Census Block Groups have a lower value for that particular column.
+    - They are normalized using [min-max normalization](https://en.wikipedia.org/wiki/Feature_scaling), which adjusts the scale of the data so that the Census Block Group with the highest value for that column is set to 1, the Census Block Group with the lowest value is set to 0, and all of the other values are adjusted to fit within that range based on how close they were to the highest or lowest value.
+1. The standardized columns are then used to calculate each of the [Justice40 candidate scores](#justice40-candidate-scores) described in greater detail below, and the results are exported to a `.csv` file in [`data/score/csv`](data/score/csv)
```

Comment:
Let's update to call these Justice40 score experiments to make it clearer the draft nature of each one, and let's remove the `### Justice40 candidate scores` section below and link to it here until we have a clearer vision for what we want to do with these experiments and how we want to communicate about them.